app/build.gradle

```diff
@@ -7,15 +7,15 @@ android {
 
     defaultConfig {
         minSdkVersion 17
-        targetSdkVersion 30
+       targetSdkVersion 31
```

Comment:
```suggestion
        targetSdkVersion 31
```

**suggestion**: Correcting wrong indentation

Reply:
The `targetSdkVersion` change was suggested by `AndroidStudio`, may be because I update it to the lastest version.

Reply:
@jfoucry I would advise to revert this change then, and see whether that works

Reply:
ok.